people/Yonatan119.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "יונתן בן עזרא",
+    "github": "yonatan119",
+    "linkedin": "https://www.linkedin.com/in/yonatan-benezra-554483174/",
+    "description": "אוהב לגרום לדמיון לעלות על המסך"
```

Comment:
We would be happy if you could tell a little about yourself, things you like to do, professional background including technological programming languages, hobbies and most importantly things that excite you :)